docs/build-on-linea/hackathons.md

```diff
@@ -101,36 +103,40 @@ Ultimately, a "high-quality" submission is quite subjective, but you can imagine
 
 #### How to submit
 
-Submitting to the hackathon takes just two steps:
+To submit to the May hackathon, just follow the [Linea builder launchpad steps](https://aspecta.id/builder-matrix/Linea-builder-launchpad)! You'll be able to receive builder NFTs, see the skills of other builders to form teams, and submit your project there!
 
-1. [Register for the current month's hackathon](https://linea.deform.cc/cook-off-april-2024)
-1. [Submit your project here](https://linea.deform.cc/cook-off-april-2024-submit).
+### May 2024 (1 ETH pool prize + $500 USD [Dune API](https://docs.dune.com/api-reference/overview/introduction) pool prize) :frog:
 
-### April 2024 (1 ETH pool prize + $1000 USD [efrog](https://twitter.com/efrogs_on_linea) pool prize) :frog:
+This month, you can win both the 1 ETH pool prize in addition to extra prize money for using the [Dune API](https://docs.dune.com/api-reference/overview/introduction). We'll be prioritizing submissions that are building a short cut via [onthis.xyz](https://onthis.xyz) or with the Dune API.
 
-This month, you can win both the 1 ETH pool prize in addition to extra prize money from the [efrog community](https://twitter.com/efrogs_on_linea).
+**Register and submit through the [Linea Builder Launch Pad](https://aspecta.id/builder-matrix/Linea-builder-launchpad)**
 
-**Register here: https://linea.deform.cc/cook-off-april-2024**
+**Due 11:59PM EST, May 31st, 2024**
 
-**Submit here: https://linea.deform.cc/cook-off-april-2024-submit/**
+#### 1 ETH pool prize
 
-**Due 11:59PM EST, April 30, 2024**
+We'll be distributing 1 ETH amongst the top qualified projects. This can take the form of 2 outstanding projects receiving 0.5 ETH each, 10 projects receiving 0.1 ETH each, or, in the case that no submissions meet our quality bar, no ETH being distributed for that month.
```

Comment:
```suggestion
We'll be distributing 1 ETH amongst the top qualified projects. This can take the form of two outstanding projects receiving 0.5 ETH each, 10 projects receiving 0.1 ETH each, or, in the case that no submissions meet our quality bar, no ETH being distributed for that month.
```